### PR TITLE
web: use webhook name in views.

### PR DESCRIPTION
**Pull request by sashaostrikov**
This commit includes changes in GraphQL layer and UI.

Follow up to https://github.com/sourcegraph/sourcegraph/pull/44756.

Closes https://github.com/sourcegraph/sourcegraph/issues/44757

Test plan:
Updated storybook and local sg run with visual checks.

### Changes
image

image

### CODE REVIEW

**Comment by sg-e2e-regression-test-bob** [comment]
## Bundle size report 📦

| Initial size | Total size | Async size | Modules |
| --- | --- | --- | --- |
| 0.00% (0.00 kb) | 0.00% (+0.11 kb) | 0.00% (+0.11 kb) | 0.00% (0) |

Look at the [Statoscope report](https://console.cloud.google.com/storage/browser/_details/sourcegraph_reports/statoscope-reports/ao/ui/webhooks/use-name/compare-report.html;tab=live_object) for a full comparison between the commits d8ffd23 and a30d5f0 or [learn more](https://docs.sourcegraph.com/dev/how-to/testing#bundlesize).


<details>
<summary>Open explanation</summary>

- `Initial size` is the size of the initial bundle (the one that is loaded when you open the page)
- `Total size` is the size of the initial bundle + all the async loaded chunks
- `Async size` is the size of all the async loaded chunks
- `Modules` is the number of modules in the initial bundle
</details>

**Review by ryanslade — APPROVED** [review]

**Comment by ryanslade** [comment]
(I only skimmed the FE changes)

File: client/web/src/site-admin/SiteAdminWebhooksPage.story.tsx [hunk 1]
```diff
@@ -112,11 +106,12 @@ export const FiveWebhooksFound: Story = () => (
 
 FiveWebhooksFound.storyName = '5 webhooks found'
 
-function createWebhookMock(kind: ExternalServiceKind, urn: string): WebhookFields {
+function createWebhookMock(name: string, kind: ExternalServiceKind, urn: string): WebhookFields {
     return {
         __typename: 'Webhook',
         createdAt: '',
         id: `webhook-${urn}`,
+        name: name,
```
Comment (mrnugget) [inline client/web/src/site-admin/SiteAdminWebhooksPage.story.tsx:114]: I'm not sure, but I suspect some linter will complain that you can use shorthand syntax here:
```suggestion
        name,
```

**Review by mrnugget — APPROVED** [review]

File: client/web/src/site-admin/SiteAdminWebhooksPage.story.tsx (hunk 1 above)
Comment (sashaostrikov) [reply client/web/src/site-admin/SiteAdminWebhooksPage.story.tsx:114]: It didn't, but I'll fix it anyway if it looks better. What exactly is this kind of magic?

Comment (sashaostrikov) [reply client/web/src/site-admin/SiteAdminWebhooksPage.story.tsx:114]: oh, it did

```
/root/buildkite/build/sourcegraph/client/web/src/site-admin/SiteAdminWebhooksPage.story.tsx
--
  | 114:9  error  Expected property shorthand  object-shorthand
```

Comment (sashaostrikov) [reply client/web/src/site-admin/SiteAdminWebhooksPage.story.tsx:114]: okay, found it: if the property name is the same as variable which is assigned to this property -- shorthand syntax can be used, neat thing.

**Pull request merged**